Storage/StorageTests/Tools/StorageTypeExtensionsTests.swift

```diff
@@ -1026,6 +1026,36 @@ final class StorageTypeExtensionsTests: XCTestCase {
         XCTAssertEqual(searchResult, storedSearchResult)
     }
 
+    func test_loadInboxNote_by_siteID_id() throws {
+        // Given
+        let inboxNote = storage.insertNewObject(ofType: InboxNote.self)
+        inboxNote.siteID = 123
+        inboxNote.id = 321
+
+        // When
+        let storedInboxNote = try XCTUnwrap(storage.loadInboxNote(siteID: 123, id: 321))
+
+        // Then
+        XCTAssertEqual(inboxNote, storedInboxNote)
+    }
+
+    func test_loadAllInboxNotes_by_siteID() throws {
+        // Given
+        let inboxNote1 = storage.insertNewObject(ofType: InboxNote.self)
+        inboxNote1.siteID = 123
+        inboxNote1.id = 321
```

Comment:
nit: maybe we can also test the sorting by descending `dateCreated` here too?

Reply:
Make sense 👍 b785782cad678aa9eff086ddcbb4cb90319cde3e